.github/actions/package.json

```diff
@@ -7,6 +7,7 @@
   "license": "MIT",
   "scripts": {
     "build": "yarn run esbuild main.ts --outdir=./dist --platform=node --bundle",
+    "build-check-team-approval": "yarn run esbuild check-team-approval.ts --outdir=./dist --platform=node --bundle",
```

Comment:
Consider consolidating all build tasks into `yarn build`, e.g. by adding `yarn build:main` and `yarn build:team-approval`